modules/ebi-metagenomics/hmmer/hmmsearch/main.nf

```diff
@@ -0,0 +1,67 @@
+
+process HMMER_HMMSEARCH {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/hmmer:3.4--hdbdd923_1' :
+        'biocontainers/hmmer:3.4--hdbdd923_1' }"
+
+    input:
+    tuple val(meta), path(fasta)
+    path hmm_db
```

Comment:
I would put this as the first input:
tuple val(meta), path(hmm)
path fasta

see comment on meta.yml